### DISCUSSION TITLE
Multiselect all

### DISCUSSION THREAD

**Discussion by huyz** [junegunn/fzf#3543] [Q&A] [answered]
With multiselect, I'd like to offer the option of selecting all or none. How can I configure fzf to provide these bindings?

I notice that now if I immediately hit ENTER (without any TAB selections), this means select only one entry.  It used to be that hitting ENTER would select none.  Either something changed or I messed up my configuration.

**Comment by LangLangBart** [comment, marked as answer]
> With multiselect, I'd like to offer the option of selecting all or none. How can I configure fzf to provide these bindings?

The action `toggle-all` accomplishes this. You can activate it by pressing <kbd>⇥ Tab</kbd> with the following command.
```bash
seq 12 | fzf --bind 'tab:toggle-all'
```

> It used to be that hitting ENTER would select none.

If you don't want the <kbd>⏎ Enter</kbd> key to print the selection and simply leave `fzf`, you can use `abort` or `become:` [^1]. See `man fzf` for details.

```bash
# exit status 130
fzf --bind 'enter:abort'
echo $?
# 130

# exit status 0
fzf --bind 'enter:become:'
echo $?
# 0
```

[^1]: [release 38](https://github.com/junegunn/fzf/releases/tag/0.38.0)

**Reply by huyz** [reply]
Thanks
